repair/repair.cc

```diff
@@ -593,6 +593,7 @@ repair_info::repair_info(repair_service& repair,
     streaming::stream_reason reason_,
     std::optional<utils::UUID> ops_uuid)
     : db(repair.get_db())
+    , rs(repair)
     , messaging(repair.get_messaging().container())
```

Comment:
If we now store a reference to repair do we still need to save all these additional references? Maybe for convenience?
Not suggesting to change in this patch.

Reply:
Yes, since after this change repair_info can access objects like db and friends using repair_serice, we can kill some of the references here. I want the change to be minimum for easier backport.